keyboard/src/modules/Keyboard.rb

```diff
@@ -1551,6 +1539,23 @@ def all_keyboards
       # eval is necessary for translating the texts needed to be translated
       content ? Builtins.eval(content) : {}
     end
+
+    def call_set_x11_keymap
+      args = [@XkbLayout, @XkbModel, @XkbVariant, @XkbOptions]
+      return if args.all?(&:empty?)
+      
+      # According to localectl syntax, empty options must be entered as "" if
+      # there are more non-empty options afterwards and skipped otherwise
```

Comment:
I had to read this several times ... and still didn't get it until I read the code ;-)